data_models/fx_settings.py

```diff
@@ -1,12 +1,83 @@
-from dataclasses import dataclass
+import sys
+
+sys.path.append('..')
+sys.path.append('data_models')
 
+import json, ntpath
+from dataclasses import dataclass
+from complex_encoder import ComplexEncoder
 
 @dataclass
 class FxSettings:
     '''FxSettings are an attribute of both Tracks and Loops.
     Only "_slip" doesn't make sense when applying effects to loops'''
-    volume: float = 0.5
-    pan: float = 0.5
-    is_reversed: bool = False
-    pitch_adjust: int = 0
-    slip: int = 0
+    _volume: float = 1
+    _pan: float = 0.5
+    _is_reversed: bool = False
+    _pitch_adjust: int = 0
+    _slip: int = 0
+
+    @property
+    def volume(self):
+        return self._volume
+    
+    @volume.setter
+    def volume(self, value):
+        self._volume = value
```

Comment:
Just wondering about this part of the class -- what's the use case for these getters and setters rather than just accessing the `volume` property?